docs/manual/glossary.md

```diff
@@ -0,0 +1,78 @@
+---
+uid: glossary
+---
+
+# Glossary
+
+## D
```

Comment:
nit: do we really need letter sections? There are < 10 entries and just one for (almost) every letter.

Reply:
I find readability much better. This also prevents the right-side menu from being one long flat list. Glossary generally have per-letter grouping/section too. I'm also hopeful we can add more definitions as needed.